src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/ProjectConfigurationStateSynchronizerTest.cs

```diff
@@ -28,19 +28,32 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer;
 public class ProjectConfigurationStateSynchronizerTest(ITestOutputHelper testOutput) : LanguageServerTestBase(testOutput)
 {
     [Fact]
-    public async Task ProjectConfigurationFileChanged_Removed_UnknownDocumentNoops()
+    public async Task ProjectConfigurationFileChanged_Removed_UntrackedProject_CallsUpdate()
```

Comment:
This is a change to the logic of the synchronizer itself, where it will call Update, but Update will no-op for a non-existent project, so holistically the behaviour is the same